scripts/functions/studies/studies.py

```
@@ -13,7 +13,7 @@
 
 
 def get_all_studies_uuid():
-    return requests.get(constant.GET_STUDIES).json()
+    return requests.get(constant.GET_SUPERVISION_STUDIES).json()
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)